tensorboard/webapp/runs/views/runs_table/runs_data_table.scss

```diff
@@ -18,12 +18,11 @@ $_circle-size: 20px;
 $_arrow_size: 16px;
 
 :host {
-  width: 100%;
-}
+  min-width: 100%;
 
-:host {
-  overflow-y: auto;
-  width: 100%;
+  & ::ng-deep tb-data-table .data-table {
```

Comment:
This replaces the previous border-bottom of the .filter-row. It ensures that the entire table has a border-top, instead of just the part touching the filter row.